Source/DafnyCore/Reporting.cs

```diff
@@ -235,6 +235,13 @@ public class ConsoleErrorReporter : BatchErrorReporter {
           msg = nestedToken.Message ?? "[Related location]";
           errorLine += $" {msg} {TokenToString(tok)}";
         }
+
+        if (DafnyOptions.O.CompileVerbose && false) { // Need to control tests better before we enable this
```

Comment:
Should this be removed?

Reply:
It was intended to be. Although it will be added back when we figure out which option to control showing explanations on the command-line.